patches/minecraft/net/minecraft/client/renderer/entity/player/PlayerRenderer.java.patch

```diff
@@ -10,6 +10,19 @@
     }
  
     public Vec3 m_7860_(AbstractClientPlayer p_117785_, float p_117786_) {
+@@ -123,6 +_,12 @@
+             if (useanim == UseAnim.SPYGLASS) {
+                return HumanoidModel.ArmPose.SPYGLASS;
+             }
++
++            HumanoidModel.ArmPose forgeArmPose = net.minecraftforge.client.ClientRegistry.getArmPose(useanim);
++            if (forgeArmPose == null) {
++               throw new RuntimeException("No ArmPose registered for UseAnim '" + useanim + "'");
++            }
++            return forgeArmPose;
```

Comment:
This should just be a generic hook that allows setting the arm pose. I.e. call `IItemRenderProperties#getArmPose` here regardless of whether the item is in use or not. Then you do not need the global map from UseAnim to ArmPos and it also allows greater flexibility.